Sources/Foundation/Process.swift

```diff
@@ -940,6 +942,13 @@ open class Process: NSObject, @unchecked Sendable {
         var spawnAttrs: posix_spawnattr_t? = nil
 #else
         var spawnAttrs: posix_spawnattr_t = posix_spawnattr_t()
+#endif
+#if os(Android)
+        guard var spawnAttrs else {
```

Comment:
This doesn't make any sense, as it is set to `nil` just 5 lines above, so this causes this code to always fail.

Maybe you needed this `guard` in some previous iteration of this patch, but it is no longer needed.

Reply:
Thanks, you're correct! It looks like I can't get rid of it yet, but the nullability on the NDK is broken for `posix_spawnattr_init`. I will follow-up with a nullability annotation change and with a change that removes that then.